.github/CONTRIBUTING.md

```diff
@@ -96,7 +97,21 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirety.
+
+Any pull-request that does not adequately complete the provided template may be closed or marked 'do not merge' by maintainers.
+ - Balance changes must indicate that they affect the balance of the game.
```

Comment:
```suggestion
 - Any changes that may affect game balance should be documented as a balance change. Bug fixes that affect balance should be marked as a balance change if the bug could reasonably be seen as an intentional feature by somebody that has no knowledge of the code.
```

Reply:
@KubeRoot @Crossedfall

Reply:
```suggestion
 - Any changes that may affect game balance should be documented as a balance change. This also applies to bug fixes which directly alter the game's balance.
```

I think it'd be too difficult to determine what a layman would and wouldn't know is a balance change in this context. Probably better to just be broad.